quickcomm/external_host_requests.py

```diff
@@ -0,0 +1,46 @@
+
+
+
+# This file deals with calling other API endpoints and returning the results in
+# a dictionary format. This file also deals with the weirdness of pagination
+# and caching external requests.
+
+import requests_cache
+
+# This caches the requests for 5 minutes, but this can be changed
+session = requests_cache.CachedSession('external_cache', expire_after=300)
+
+def get_authors(url):
+    """Get all author details from a host. This function will raise an exception
+    if the request fails."""
+
+    # clean up the URL
+    if url[-1] == '/':
```

Comment:
is there a cleaner way to do this? not important but it seems odd having to clean URLs with this